src/types.rs

```diff
@@ -10,17 +10,16 @@ pub type Place = Vec<usize>;
 
 /// A symbol for an unspecified term. Only carries meaning alongside a [`Signature`].
 ///
-/// To construct an [`Variable`], use [`Signature::new_var`]
+/// To construct an `Variable`, use [`Signature::new_var`]
```

Comment:
"an `Variable`" -> "a `Variable`"

Complete sentences (i.e. first letter capitalized, period at the end) also make the rendered docs easier to read. I'll note that here, but please correct throughout.

``To construct a `Variable`, use [`Signature::new_var`].``